data_diff/database.py

```diff
@@ -173,7 +272,9 @@ def __init__(self, host, port, database, user, password, *, thread_count):
     def create_connection(self):
         postgres = import_postgres()
         try:
-            return postgres.connect(**self.args)
+            c = postgres.connect(**self.args)
+            # c.cursor().execute("SET TIME ZONE 'UTC'")
```

Comment:
I assume you just used this to debug? I think it's fair game for us to change settings on the connection, as long as they don't affect the whole database. It's so much easier to normalize everything to UTC per session if that's easier, than try to wrangle equality otherwise 🤯